example/package.json

```diff
@@ -12,7 +12,7 @@
     "expo": "~22.0.0",
     "prop-types": "^15.5.10",
     "react": "16.0.0-beta.5",
-    "react-native": "~0.49.3"
+    "react-native": "https://github.com/expo/react-native/archive/sdk-22.0.1.tar.gz"
```

Comment:
is this changed to support `ListView` ? because for `react-native@0.49` is Deprecated right?

Reply:
@hhg2288 this expo version is equivalent to rn `0.49.3`. this change is not related to listview, migrating to flatlist can come in a future commit.